checkers/bool.go

```diff
@@ -104,14 +104,41 @@ type deepEqualsChecker struct {
 //
 // This checker differs from gocheck.DeepEquals in that
 // it will compare a nil slice equal to an empty slice,
-// and a nil map equal to an empty map.
-var DeepEquals gc.Checker = &deepEqualsChecker{
-	&gc.CheckerInfo{Name: "DeepEquals", Params: []string{"obtained", "expected"}},
-}
+// and a nil map equal to an empty map and that
+// it will panic if called upon to compare unexported fields.
```

Comment:
Changing the behavior of unexported fields like this seems a little too drastic a change to me. perhaps leave DeepEquals alone and generally use CmpEquals for new tests.